app/build.gradle

```diff
@@ -61,14 +68,11 @@ dependencies {
     compile 'com.android.support:multidex:1.0.1'
     compile 'com.google.zxing:javase:3.3.0'
     compile 'com.google.zxing:core:3.3.0'
-
+    compile 'com.android.support.constraint:constraint-layout:1.0.2'
```

Comment:
Could you remove this dependency, it is not working for me but everything works fine once I remove it...

Could you also put the `androidTestCompile` back to where it was originally placed 😇